### PR TITLE
[Compression] pruning speedup: support input/output masks

### PR DESCRIPTION
**Pull request by J-shang**
### Description ###
Support input/output masks in pruning speedup, more test cases are needed.

#### Test Options ####
  - [x] fast test
  - [ ] full test - HPO
  - [ ] full test - NAS
  - [x] full test - compression

### Checklist ###
  - [x] test case
  - [ ] doc

### CODE REVIEW

File: nni/compression/pytorch/speedup/v2/model_speedup.py [hunk 1]
```diff
@@ -211,6 +212,8 @@ def propagate_originally(self):
             self.node_infos[node].output_origin = output
             self.node_infos[node].output_inplace = \
                 tree_map_zip(lambda t: t.clone().detach() if isinstance(t, torch.Tensor) else deepcopy(t), output)
+            self.node_infos[node].output_masks = \
+                tree_map_zip(lambda t: torch.ones_like(t).clone().detach() if isinstance(t, torch.Tensor) else None, output)
```
[the remark this answers is not in the document]
Comment (J-shang) [reply nni/compression/pytorch/speedup/v2/model_speedup.py:216]: in fact, if only have one pytree as input, `tree_map_zip` will directly call `tree_map`

**Review by super-dainiu — APPROVED** [review]

**Review by Bonytu — APPROVED** [review]
